tests/test_ingest.py

```diff
@@ -181,9 +190,11 @@ def test_ingest_local_dense_matrix(self):
         """Ingest Pipeline should extract and transform local dense matrices
         """
 
+        study_accession = get_random_study_accession()
+
         args = [
```

Comment:
It looks like you have 2 set of args here. Is that to be as aspected?

Reply:
Per our chat, there is actually only one set of arguments for each test, but the GitHub diff UI makes it look like two.